Second

kensayo opened the pull request:
Code review for Milestone 1 & 2

memelopez committed:
making branch cover

memelopez committed:
HTML for 'cover' section

memelopez committed:
add changes to navbar buy pulling from second

memelopez committed:
finish with 'cover' section HTML and CSS for mobile

kensayo committed:
Added content for section

Added part of the content for latest news section

memelopez committed:
fixed minor details in cover

kensayo committed:
Added all the content

Added the content from latest news into HTML file

memelopez committed:
fix linter errors

kensayo committed:
Minor bug fixes for latest news

memelopez committed:
fix linter errors2

memelopez committed:
fix linter errors3

memelopez committed:
fix linter errors4

memelopez committed:
fix linter errors5

kensayo committed:
Added gitignore file

memelopez committed:
fix linter errors6

kensayo committed:
Fixed CSS linters

memelopez committed:
Merge pull request #7 from memelopez/cover

add Cover mobile

kensayo committed:
Fixed conflicts

kensayo committed:
Fixed linters

kensayo committed:
Merge pull request #8 from memelopez/latest-news

Latest news merge

memelopez committed:
fix linter errors7

kensayo committed:
Add content for latest deals

Add the content for latest news and apply style

kensayo committed:
Adding images for latest deals

kensayo committed:
Merge pull request #9 from memelopez/latest-deals

Latest deals

kensayo committed:
Updated README

Updated readme file and added first screen shot

memelopez committed:
conclude merge: cover si up to date and has mobile styles only

kensayo committed:
Added footer section

Added the skeleton for footer section

kensayo committed:
Added icons

Added the icons for social media in the footer

kensayo committed:
Added footer caption

kensayo committed:
Fixed linters

kensayo committed:
Fixed linters

Minors linters errors fixed

kensayo committed:
Lintern error

kensayo committed:
Merge pull request #11 from memelopez/footer

Footer

layzrd reviewed (changes requested):
### Status: Require changes 🌳 ♲
Hi @memelopez and @kensayo, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏   I appreciate your hard work in replicating thenextweb.com. Kindly fix the following issues. 

[REQUIRED]

> Small screens

- [ ] Fix broken links from the readme
- [ ] Add spacing between components
- [x] Add the latest funding section
- [x] Add the deals section
- [x] Add the footer section

> General

- [ ] Make sure the page has a similar look as the original page
- [ ] Replicate all sections from the original page

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

memelopez committed:
finish 'catalog' section mobile

memelopez committed:
conclude merge: add catalog to everything else

memelopez committed:
fix linters

memelopez committed:
fix linters2

memelopez committed:
fix linters3

memelopez committed:
Merge pull request #12 from memelopez/catalog

finished with Catalog mobile layout

kensayo committed:
Added media queries for tablet

Had to rebuild the structure of navbar also added the media queries for the navbar with all the menu

kensayo committed:
Add fonts & fix nabvar

Added avalaon and shantex font's, fixed nabvar with menu

kensayo committed:
Fixed linters

kensayo committed:
Merge branch 'second' into navbar-responsive

kensayo committed:
Merge pull request #13 from memelopez/navbar-responsive

Navbar responsive - Still need to fix 2 linters error, don't know what's happening.

kensayo committed:
Adding flex for latest section

kensayo committed:
Complete responsive

Responsive design for latest sections

kensayo committed:
Fixed linters

kensayo committed:
Merge pull request #14 from memelopez/latest-section-responsive

Latest section responsive

memelopez committed:
finish 'funding' section for mobile layout

memelopez committed:
conclude merge: add funding section to all of Kenny's wrok

memelopez committed:
fix linter

memelopez committed:
add screenshot to README

memelopez committed:
Merge pull request #15 from memelopez/catalog

add 'funding' section to second

memelopez committed:
done with cover. it's  responsive all the way

memelopez committed:
done with cover. it's  responsive all the way2

memelopez committed:
Merge pull request #16 from memelopez/cover

Cover is full responsive

kensayo committed:
Fixed linters in font.css

kensayo committed:
Fixed navbar

Navbar stick in the top of the viewport

kensayo committed:
Added icon for universal access

kensayo committed:
Fixed navbar text color

kensayo committed:
Updated access icon

Added transition to icon and modified the source

kensayo committed:
Fixed linters

kensayo committed:
Merge pull request #17 from memelopez/icon-access

Icon access

kensayo committed:
Revert "Fixed navbar"

This reverts commit abda9cac344d320ff5a8bd657f4d4608c8fd66ab.

memelopez committed:
'funding' and 'catalog' sections are full responsive

kensayo committed:
Merge branch 'second' of https://github.com/memelopez/microv-wk2proj into second

memelopez committed:
conclude merge: add full responsive 'funding' and 'catalog' sections to Kenny's work

memelopez committed:
fix linters

memelopez committed:
Merge pull request #18 from memelopez/catalog

'catalog' and 'funding' sections are full responsive and joining to Kenny's work

kensayo committed:
Fixed top nabvar

kensayo committed:
Fixed README file

memelopez committed:
fixed minor details

memelopez committed:
fix a detail with the lower right blue button

memelopez committed:
fix broken links in README

memelopez committed:
fix details in README
